### PR TITLE
Test8 is moved to Page Object

### LINKED ISSUE #17: Test#8 is moved to Page Object

### PR DESCRIPTION
**Pull request by lvrub**
@extsoft please review my Test#8

Closes #17

### CODE REVIEW

File: src/main/java/net/commments/object/BCommentTable.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    public BCommentTable(WebDriver webDriver) {
+        this.driver = webDriver;
+    }
+
+    public void checkCommentInTable(int numberComment) {
+        this.driver.findElements(By.name("SelectedId")).get(numberComment).click();
+    }
+
+
+    public String checkActiveStatus() {
```
Comment (extsoft) [inline src/main/java/net/commments/object/BCommentTable.java]: Do you check something in this method? I don't see. That's why please rename it to `commentStatus` or something similar.

File: src/main/java/net/commments/ft_noPageObject/Test8.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package net.commments.ft_noPageObject;
```
[the remark this answers is not in the document]
Comment (lvrub) [reply src/main/java/net/commments/ft_noPageObject/Test8.java]: @extsoft 

done,  pushed

File: src/main/java/net/commments/ft_noPageObject/Test4.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package net.commments.ft_noPageObject;
```
[the remark this answers is not in the document]
Comment (lvrub) [reply src/main/java/net/commments/ft_noPageObject/Test4.java]: @extsoft 

done,  pushed

File: src/main/java/net/commments/object/BCommentListActions.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import static org.testng.Assert.assertEquals;
+
+public class BCommentListActions implements CommentListActions {
+    public final WebDriver driver;
+
+    public BCommentListActions(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void openActionList() {
```
[the remark this answers is not in the document]
Comment (lvrub) [reply src/main/java/net/commments/object/BCommentListActions.java]: @extsoft 

Just to check elements of drop-down

File: src/main/java/net/commments/object/BCommentListStatus.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            assertEquals(text3, "Inactive");
+
+        }
+
+    }
+
+    public void selectStatus(String status) {
+        new Select(this.driver.findElement(By.id("SelectedStatus"))).selectByVisibleText(status);
+        this.driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
+
```
[the remark this answers is not in the document]
Comment (lvrub) [reply src/main/java/net/commments/object/BCommentListStatus.java]: done

File: src/main/java/net/commments/object/BCommentTable.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    private final WebDriver driver;
+
+    public BCommentTable(WebDriver webDriver) {
+        this.driver = webDriver;
+    }
+
+    public void checkCommentInTable(int numberComment) {
+        this.driver.findElements(By.name("SelectedId")).get(numberComment).click();
+    }
+
```
[the remark this answers is not in the document]
Comment (lvrub) [reply src/main/java/net/commments/object/BCommentTable.java]: done

File: src/main/java/net/commments/object/BCommentTable.java (hunk 1 above)
Comment (lvrub) [reply src/main/java/net/commments/object/BCommentTable.java]: done

File: src/main/java/net/commments/object/BCommentListStatus.java [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+import static org.testng.Assert.assertEquals;
+
+public class BCommentListStatus implements CommentListStatus {
+    public final WebDriver driver;
+
+    public BCommentListStatus(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void openStatusList() {
```
[the remark this answers is not in the document]
Comment (lvrub) [reply src/main/java/net/commments/object/BCommentListStatus.java]: Just to check elements of drop-down

File: src/main/java/net/commments/object/BCommentListActions.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (extsoft) [reply src/main/java/net/commments/object/BCommentListActions.java]: Why do you check them? Is it defined in the test?

File: src/main/java/net/commments/object/BCommentListStatus.java (hunk 7 above)
[the remark this answers is not in the document]
Comment (extsoft) [reply src/main/java/net/commments/object/BCommentListStatus.java]: @lvrub Why do you check them?

File: src/main/java/net/commments/object/BCommentTable.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+//    public String commentStatus() {
```
Comment (extsoft) [inline src/main/java/net/commments/object/BCommentTable.java]: Please remove commented code or uncomment it.

File: src/main/java/net/commments/object/CommentTable.java [hunk 9]
```diff
@@ -0,0 +1,11 @@
+package net.commments.object;
+
+public interface CommentTable {
+
+    void checkCommentInTable(int numberComment);
+
+//   String commentStatus();
```
Comment (extsoft) [inline src/main/java/net/commments/object/CommentTable.java]: Please remove commented code or uncomment it.

File: src/main/java/net/commments/object/BCommentListStatus.java (hunk 7 above)
[the remark this answers is not in the document]
Comment (extsoft) [reply src/main/java/net/commments/object/BCommentListStatus.java]: @lvrub Why do you check them?

File: src/main/java/net/commments/object/BCommentListActions.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (extsoft) [reply src/main/java/net/commments/object/BCommentListActions.java]: @lvrub Why do you check them? Is it defined in the test?

File: src/main/java/net/commments/object/CommentTable.java (hunk 9 above)
Comment (lvrub) [reply src/main/java/net/commments/object/CommentTable.java]: done

File: src/main/java/net/commments/object/BCommentListStatus.java (hunk 7 above)
[the remark this answers is not in the document]
Comment (lvrub) [reply src/main/java/net/commments/object/BCommentListStatus.java]: removed

File: src/main/java/net/commments/object/BCommentListActions.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (lvrub) [reply src/main/java/net/commments/object/BCommentListActions.java]: > Preview

removed

**Pull request merged**